config/site_settings.yml

```diff
@@ -2373,10 +2373,6 @@ tags:
     client: true
     default: 5
     min: 1
-  show_filter_by_tag:
```

Comment:
After merging please let me know so that I can write a migration to drop this site setting from the database.